src/main/java/com/xebia/models/Register.java

```diff
@@ -0,0 +1,63 @@
+package com.xebia.models;
+
+import java.time.LocalDate;
+import java.util.*;
+
+public class Register {
+
+  private UUID userId;
+  private UUID bookId;
+  private LocalDate rentDate;
+  private LocalDate returnDate;
+
+  public Register(UUID userId, UUID bookId, LocalDate rentDate) {
+    this.userId = userId;
+    this.bookId = bookId;
+    this.rentDate = rentDate;
+  }
+
+  public UUID getUser_id() {
```

Comment:
```suggestion
  public UUID getUserId() {
```

I should update it to that name. Same for the rest of methods

Reply:
sorry, i missed some of them, everything corrected